app/bundles/CoreBundle/Translations/en_US/messages.ini

```diff
@@ -426,6 +431,7 @@ mautic.core.update.index="Update Mautic"
 mautic.core.update.migrating.database.schema="Migrating database schema..."
 mautic.core.update.no_cache_data="Could not read the cached update data to apply the update."
 mautic.core.update.now="Update Now"
+mautic.core.update.composer="You have Composer updates enabled. This means that you can only update Mautic through the Command Line. Read more in the <a target='_blank' href='https://mau.tc/TODO'>documentation</a>."
```

Comment:
Will need to create Mautic docs page for this

Reply:
Created short URL for this, will create the actual content on Monday evening CEST: https://mau.tc/switch-to-composer